src/main/java/uk/gov/hmcts/reform/waworkflowapi/clients/service/CamundaClient.java

```diff
@@ -8,24 +8,32 @@
 import uk.gov.hmcts.reform.waworkflowapi.clients.model.DmnValue;
 import uk.gov.hmcts.reform.waworkflowapi.clients.model.EvaluateDmnRequest;
 import uk.gov.hmcts.reform.waworkflowapi.clients.model.SendMessageRequest;
+import uk.gov.hmcts.reform.waworkflowapi.config.CamundaFeignConfiguration;
 
 import java.util.List;
 import java.util.Map;
 
 @SuppressWarnings("PMD.UseObjectForClearerAPI")
 @FeignClient(
     name = "camunda",
-    url = "${camunda.url}"
+    url = "${camunda.url}",
+    configuration = CamundaFeignConfiguration.class
 )
 public interface CamundaClient {
 
     String SERVICE_AUTHORIZATION = "ServiceAuthorization";
 
-    @PostMapping(value = "/message", produces = MediaType.APPLICATION_JSON_VALUE)
+    @PostMapping(
+        value = "/message",
+        consumes = MediaType.APPLICATION_JSON_VALUE,
+        produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
not required.

Reply:
Hi @indvamshi it was required to add the consume as by default the content type was:
x-www-form-urlencoded as you can see it now inherits from the CamundaFeignConfiguration.class